src/components/index.js

```diff
@@ -1,6 +1,8 @@
 import Header from "./Header/Header";
+import QuizStart from "./StartQuizPage/index.jsx";
```

Comment:
We won't need this `index.js` file, you can delete it. We will import from "./components/componentName" like Vaibhav and Kashmeera are doing